src/apps/chifra/pkg/utils/client.go

```diff
@@ -0,0 +1,7 @@
+package utils
+
+import "strings"
+
+func IsClientErigon(version string) bool {
```

Comment:
Add a TODO item saying something like "Add queries for other node software (besu, nethermind, geth, etc." We will eventually need it anyway, I'm sure.